client/my-sites/domains/domain-management/name-servers/wpcom-nameservers-toggle.jsx

```diff
@@ -25,13 +26,18 @@ class NameserversToggle extends PureComponent {
 				</span>
 
 				<form className="name-servers__toggle">
-					<ToggleControl
-						id="wp-nameservers"
-						name="wp-nameservers"
-						onChange={ this.handleToggle }
-						checked={ this.props.enabled }
-						value="active"
-					/>
+					<FormLabel>
+						<ToggleControl
+							id="wp-nameservers"
```

Comment:
Something to note here (and I realize it's pre-existing) is that the `id` attribute never makes it further, because the `ToggleControl` doesn't do anything with it.